build/release/README.md

```diff
@@ -0,0 +1,62 @@
+# lnd's Reproducible Build System
+
+This package contains the container that the `lnd` project uses in order to
+build binaries for each new release. As go Go 1.13, with some new build
+flags, binaries are now reproducible, meaning to developers can build the
+binary on distinct machines, and end up with a byte-for-byte identical
+binary. However, this wasn't _fully_ solved in Go 1.13, as the build system
+still includes the directory the binary is built into the binary itself,.
+This means that in order for developers/users to produce this binary, it
+needs to be built from the _same directory_.
+
+## Building a New Release
+
+### MacOS/Windows
+
+The first requirement is to have [`docker`](https://www.docker.com/)
+installed locally and running.
+
+Next, from the top-level `lnd` directory, run the following command:
```

Comment:
Needs to say:
> Next, change into the `build/release` directory and run the following command:

Otherwise it's not going to work. The `docker run` command shown later also expects you to be in `build/release`.